### PR TITLE
Feature/create styles base files

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- #1 
- Create base file styles

### CODE REVIEW

File: practices/practice-one/src/styles/common/index.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    height: 100vh;
+    background-image: url(../../assets/images/backgound.jpeg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    font-family: var(--font-roboto);
+    overflow: hidden;
+}
+
+.container {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/common/index.css]: Make this as a component

File: practices/practice-one/src/styles/variables.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+:root {
+    --font-roboto: 'Roboto', sans-serif;
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/variables.css]: Naming not good, this so hard to re-use

File: practices/practice-one/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/node_modules
```
Comment (cunguyen-agilityio) [inline practices/practice-one/.gitignore]: should add `build` folder into this file

File: practices/practice-one/src/styles/common/index.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.app {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/common/index.css]: Why app in common?

Comment (DuongPham-dev) [reply practices/practice-one/src/styles/common/index.css]: -  I will add it to components/default.css

Comment (cunguyen-agilityio) [reply practices/practice-one/src/styles/common/index.css]: Please help to create a ticket for it, then keep in here

Comment (DuongPham-dev) [reply practices/practice-one/src/styles/common/index.css]: I will fix this via #32

**Pull request merged**